build.sh

```diff
@@ -4,15 +4,22 @@ source `dirname $0`/common.sh
 
 PKG=$1
 
-fold_start ruby19 "Installing Ruby 1.9.1"
```

Comment:
I don't know if Travis team has a preference between Ruby and Bash, so I made changes to both keeping two commits, any one of which could be cherry-picked per your preference.
Furthermore, @meatballhat has made this change which is available in #6.